s4/clients/s3.py

```diff
@@ -20,6 +22,13 @@
 
 S3Uri = collections.namedtuple("S3Uri", ["bucket", "key"])
 
+# add support for specifying a certificate to validate the SSL connection
+ssl_cert = os.environ.get('REQUESTS_CA_BUNDLE', '')
+verifySSL = True
+if ssl_cert != '':
+    verifySSL = ssl_cert
+if ssl_cert == 'IGNORE':
+    verifySSL = False
```

Comment:
it's generally not advisable to have code in the global context. This should be passed down as an argument to `get_s3_client` all the way from `sync.py`